hq_superset/__init__.py

```diff
@@ -29,6 +29,8 @@ def flask_app_mutator(app):
     # it. So I've run out of better options. (Norman 2024-03-13)
     os.environ['AUTHLIB_INSECURE_TRANSPORT'] = '1'
 
+    register_commands(app)
```

Comment:
Instead of defining the command in two different places, you can use the same pattern that this `flask_app_mutator()` function uses to add views. For example ...

```suggestion
    from . import commands  # Append to import at the start of the function

    app.cli.add_command(commands.subscribe_datasources)
```

Then you can drop the `register_commands()` function below, and move the command definition, with decorators and docstring, to the `commands` module. e.g.
```python
# commands.py
import click


@click.command("subscribe-datasources")
@click.argument("superset_base_url")
@click.argument("hq_base_url")
@click.argument("data_sources_file_path", type=click.Path(exists=True))
@click.argument("username")
@click.argument("apikey", envvar="HQ_APIKEY")
def subscribe_datasources(
    superset_base_url,
    hq_base_url,
    data_sources_file_path,
    username,
    apikey,
):
    """
    This command takes all data sources in the data_sources_file_path
    and subscribes to their changes on HQ.

    :param superset_base_url: The base URL of the superset instance. This is
        used to construct the callback URL for publishing the changes to.

    :param hq_base_url: The base URL of the HQ server containing the data
        sources you'd like to subscribe to.

    :param data_sources_file_path: Path to file containing JSON-formatted
        data source IDs by domain. Example JSON:
        {"domain": ["data_source_id"]}

    :param username: An HQ superuser username.

    :param apikey: An HQ superuser API key. Alternatively, you can set the
        HQ_APIKEY environment variable to avoid passing this as an argument.
    """
    # params separated by blank lines for better readability on CLI:
    #     $ superset subscribe-datasources --help

```